vars/centralPipelineLoad.groovy

```diff
@@ -0,0 +1,41 @@
+import com.sap.piper.Utils
+
+/**
+ * centralPipelineLoad
+ * Load a central pipeline.
+ *
+ */
+def call(Map parameters = [:]) {
+
+    handlePipelineStepErrors (stepName: 'centralPipelineLoad', stepParameters: parameters) {
+
+        def utils = new Utils()
+
+        // The coordinates of the central pipeline script
+        def repo = utils.getMandatoryParameter(parameters, 'repoUrl', null)
+        def branch = utils.getMandatoryParameter(parameters, 'branch', 'master')
+        def path = utils.getMandatoryParameter(parameters, 'jenkinsfilePath', 'Jenkinsfile')
```

Comment:
"Jenkinsfile" is a widespread naming convention for the file holding the pipeline script. Nevertheless I'm not sure if we should use that naming convention for the pipeline step here as we do it with "jenkinsfilePath". Basically it is a script describing the build control. Maybe "scriptPath" or simply "path" - if sufficiently clear in the context of the step name -  would be an option.

Reply:
I agree that "path" is sufficient and a person using the step should understand it.